.github/workflows/recipe.yaml

```diff
@@ -9,8 +9,6 @@ jobs:
 
   recipe:
 
-    name: Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏why remove the name? it is interesting in the report. Juste replace the content:

```
name: Flex recipe (PHP ${{ matrix.php }}, Sylius ${{ matrix.sylius }})
```